docs/source/tuning.rst

```diff
@@ -125,6 +125,60 @@ Experimenting with this value yields a better result at ``2e2``.
 Here, the scan matcher used rotation to still slightly mess up the result though.
 Setting the ``rotation_weight`` to ``4e2`` leaves us with a reasonable result.
 
+Special Cases
+-------------
+
+The default configuration and the above tuning steps are focused on quality.
+Only after we have achieved good quality, we can further consider low latency tasks and other special cases.
```

Comment:
Define 'low latency'

Reply:
Put this information first and mention you're talking about low latency later?

Reply:
Done